DDDEastAnglia/Views/Account/_ExternalLoginsPartial.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
```

Comment:
They are both different views of the same model, used for different purposes. They have a differing HTML structure, so it would be messy to combine the two.